src/app/public/modules/action-hub/action-hub.component.spec.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestModule
+} from '@skyux-sdk/builder/runtime/testing/browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  SkyActionHubComponent
+} from './action-hub.component';
+
+import {
+  Component,
+  Input
+} from '@angular/core';
+
+import {
+  Observable, of
+} from 'rxjs';
+
+import {
+  Configuration
+} from './types/configuration';
+
+import {
+  SkyActionHubModule
+} from './action-hub.module';
+
+// Test component
+
+@Component({
+  selector: 'skyux-action-hub-test',
+  template: `
+    <sky-action-hub
+      [config]="configAsync"
+    ></sky-action-hub>
+  `
+})
+class ActionHubTestComponent {
+  @Input()
+  public configAsync: Observable<Configuration>;
+}
+
+// end Test component
+
+describe('Action hub component', () => {
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        SkyAppTestModule,
```

Comment:
We typically don't use `SkyAppTestModule` when testing our libraries. We create a fixtures module that then pulls in just what is needed. We do this to ensure that our module for this given component is pulling in everything that it needs. The `SkyAppTestModule` pulls in everything and so it can give us a false sense of security as it pulls everything in that a SPA would pull in - but these components could be used outside of a SPA so we need to ensure we have everything pulled in correctly.

See https://github.com/blackbaud/skyux-layout/blob/master/src/app/public/modules/description-list/fixtures/description-list-fixtures.module.ts for an example.